data/src/main/java/io/gnosis/data/repositories/TokenRepository.kt

```diff
@@ -23,7 +23,7 @@ class TokenRepository(
             else
```

Comment:
This logic (replacing null token address with etherum info) should be implemented by the client gateway, shouldn't it?